### ISSUE TITLE
Revision jmx-collectd

### ISSUE THREAD

**Issue by edevosc2c** [georchestra/helm-georchestra#71]
https://github.com/georchestra/helm-georchestra/blob/4fdce64ab314a0aabe520d3ee264d681058deca0/templates/geoserver/geoserver-deployment.yaml#L237

- [ ] May not be suitable that the docker image is hosted at camptocamp.

Related https://github.com/georchestra/georchestra/issues/4423

**Comment by pmauduit** [comment]
indeed, and maybe there could be more relevant "cloud-native-shiny" solutions than using collectd. This could be an agent or a standalone server which could run as a sidecar, see https://github.com/prometheus/jmx_exporter

The repository which hosts the docker image is public though: https://github.com/camptocamp/docker-jmx-collectd
